rest_log/components/service.py

```diff
@@ -68,7 +68,7 @@ def _dispatch_with_db_logging(self, method_name, *args, params=None):
         log_entry = self._log_call_in_db(
             self.env, request, method_name, *args, params=params, result=result
         )
-        if log_entry:
+        if log_entry and isinstance(result, dict):
```

Comment:
this has been fixed in v14 already (I think we lost it in v13) https://github.com/OCA/rest-framework/commit/4e78558a1ec4fdd77df35017d2f09f2b68d337d9
Please cherry-pick this commit instead.